lib/pi_piper/bcm2835.rb

```diff
@@ -36,7 +36,8 @@ def self.pin_input(pin)
     end
 
     def self.pin_set(pin, value)
-      File.open("/sys/class/gpio/gpio#{pin}/value", 'w') {|f| f.write("#{value}") }
+      raise PiPiper::PinError, "Pin #{pin} not exported" unless self.exported?(pin)
```

Comment:
I think you should be able to just to `exported?(pin)`, without needing the `self.` in front. I haven't tested this myself, but I believe it should work that way since the code is already in that context.